check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
```

Comment:
We probably need `MISSING=` here,
or this could be a trouble:

```sh
MISSING=not-exist
export MISSING
./check.sh
```

Reply:
Good idea, thanks.